crates/channel/src/message_queues.rs

```diff
@@ -123,6 +123,7 @@ impl<K: Eq + Hash + Clone, T> PerKeyQueue<K, T> {
             // For example, many of our queues have a max capacity of 1024. To
             // handle a single rpc from a transient peer, we would end up
             // allocating ~ 96 b * 1024 ~ 64 Kib per queue.
+            //TODO: Reconsider this. Maybe true for VFN but not Validators
```

Comment:
nit: space after `//` (I think this is a convention in our codebase, although, it's annoying the formatter doesn't enforce it 😢). There's a bunch of them that need to change.